wire-ios/Wire-iOS Tests/ConversationMessageCell/XCTestCase+VerifyMessage.swift

```diff
@@ -32,7 +32,20 @@ extension ConversationMessageContext {
                                                                        spacing: 0)
 }
 
-extension XCTestCase {
+class ConversationMessageSnapshotTestCase: ZMSnapshotTestCase {
+
+    var userSession: UserSessionMock!
+
+    override func setUp() {
+        super.setUp()
+        userSession = UserSessionMock()
+    }
+
+    override func tearDown() {
+        userSession = nil
+        super.tearDown()
+    }
+
```

Comment:
how is this related to the PR?

it should be moved to own file, no?

Reply:
The cherry-pick action has added this, it's not present in the original PR: https://github.com/wireapp/wire-ios/pull/672/files

I will revert it.
We should really change the cherry-pick behaviour.

Reply:
done @netbe

Reply:
@caldrian maybe redo the cherry-pick from develop. the action does strategy --theirs.
I have added a ticket for the behaviour

Reply:
I have ensured now, that the changes match.